cpp/src/rolling/range_window_bounds.cpp

```diff
@@ -45,7 +46,9 @@ struct range_scalar_constructor {
       static_cast<duration_scalar<T> const&>(range_scalar_));
   }
 
-  template <typename T, CUDF_ENABLE_IF(std::is_integral_v<T> && not cudf::is_boolean<T>())>
+  template <typename T,
+            CUDF_ENABLE_IF(std::is_floating_point_v<T> ||
```

Comment:
I believe that this needs to be removed.

Reply:
You are right. I'll remove this shortly.

Reply:
Removed.